pkg/kubelet/container_manager_unsupported_test.go

```diff
@@ -0,0 +1,72 @@
+// +build !linux
```

Comment:
I wonder if it is possible to share this file with both linux and non linux builds, by removing the `+build tag`? That way we can avoid duplicating this logic in the `container_manager_test.go` file.